pkgs/development/libraries/proj/default.nix

```diff
@@ -11,19 +20,34 @@ stdenv.mkDerivation rec {
     sha256 = "0mymvfvs8xggl4axvlj7kc1ksd9g94kaz6w1vdv0x2y5mqk93gx9";
   };
 
+  postPatch = lib.optionalString (version == "7.2.1") ''
+    substituteInPlace CMakeLists.txt \
+      --replace "MAJOR 7 MINOR 2 PATCH 0" "MAJOR 7 MINOR 2 PATCH 1"
+  '';
+
   outputs = [ "out" "dev"];
 
-  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  nativeBuildInputs = [ cmake pkg-config ];
```

Comment:
@dotlambda pkg-config file (proj.pc) not installed when using CMake (fixed in 8.0.0, https://github.com/OSGeo/PROJ/pull/2547). That's problem, since proj is used by many projects (and out of nixpkgs) and many of them use pkg-config, not cmake.